repositories/user.js

```diff
@@ -5,9 +5,22 @@ async function getUserByEmail(email) {
 }
 
 async function createUser(userData) {
-  const { email, password, role } = { ...userData }
+  const {
+    email,
+    name,
+    surname,
+    department,
+    salary,
+    stack,
+    password,
+    role
+  } = { ...userData }
 
-  return await User.create({email: email, password: password, role: role})
+  return await User.create({email, name, surname, department, salary, stack, password, role})
+}
+
+async function getAllUsers() {
+  return await User.find({}).exec()
```

Comment:
Prepare here for pagination. Because getAllUsers will return all n number of users.
You will need skip and limit.